R/adherence.R

```diff
@@ -0,0 +1,21 @@
+#' Endpoint to gather user adherence from Bridge
+#' 
+#' @param study_id the Bridge Study ID
+#' @param user_id the Bridge User ID
+#' 
+#' @return response from Bridge adherence/activityStreams endpoint
+#' 
+#' @export
+get_adherence <- function(study_id, user_id){
+    if (is.null(study_id) || is.null(user_id)) {
+        stop(glue::glue("You need to fill both study_id and user_id"))
+    }
+    tryCatch({
+        bridgeGET(
+            glue("/v5/studies/{study_id}/participants/{user_id}/adherence/eventstream"))
+    }, error = function(e){
+        stop(glue::glue("Adherence from following ",
```

Comment:
Why can't we use the exception from Bridge since the only thing this function is doing is making an API request? Is the text included with a regular exception not helpful in this case?

Reply:
changed to e$message return